wdl/MainVcfQc.wdl

```diff
@@ -16,6 +16,7 @@ workflow MainVcfQc {
     String? bcftools_preprocessing_options
     File? ped_file
     File? list_of_samples_to_include
+    File? sample_renaming_json
```

Comment:
Should make this an option in all workflows that call MainVcfQc

Reply:
Done. I omitted Module07MinGqStep2MergePCRStatus because a) I'm suspicious of it, and b) it doesn't have the option to run per-sample comparisons anyway.